XRTK-Core/Packages/com.xrtk.core/Utilities/GizmoUtilities.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEngine;
+#endif
+
+namespace XRTK.Utilities
+{
+    /// <summary>
+    /// Gizmo drawing utilities.
+    /// </summary>
+    public static class GizmoUtilities
+    {
+#if UNITY_EDITOR
+
+        /// <summary>
+        /// Renders a capsule wire gizmo.
+        /// </summary>
+        /// <param name="position">Center position of the capsule.</param>
+        /// <param name="rotation">Center rotation of the capsule.</param>
+        /// <param name="radius">Capsule radius.</param>
+        /// <param name="height">Capsule height.</param>
+        /// <param name="color">Optional color override, will use <see cref="Gizmos.color"/> by default.</param>
+        public static void DrawWireCapsule(Vector3 position, Quaternion rotation, float radius, float height, Color color = default)
```

Comment:
I believe there's already an extension method class where we can put this. One minute while I look.

Reply:
`MixedRealityInspectorUtility` which isn't very descriptive. I'm down to renaming it and making it more specific.

Reply:
Alright, cool! I'll move it there and create a separate PR for it. If I can come up with a nice name I'll let you know :)